### PR TITLE
feat(metrics): add timing metrics to the statsd collector

### PR DESCRIPTION
**Pull request by vladikoff**
TODO:
- [x] Lock this to login and registration only

Fixes https://github.com/mozilla/fxa-content-server/issues/2285
Fixes https://github.com/mozilla/fxa-content-server/issues/2284

### CODE REVIEW

File: server/lib/statsd-collector.js [hunk 1]
```diff
@@ -157,6 +165,7 @@ StatsDCollector.prototype = {
         body.events.forEach(function (event) {
           if (event.type) {
             self.increment(event.type, tags);
+            self.timing(event.type, event.offset, tags);
```
Comment (vladikoff) [inline server/lib/statsd-collector.js]: going to add Login / Registration allow list here.

**Comment by vladikoff** [comment]
Example: 
image

@shane-tomlinson r?

**Comment by shane-tomlinson** [comment]
@vladikoff - is this still a WIP?

**Comment by vladikoff** [comment]
> @vladikoff - is this still a WIP?

no, ready for review

**Comment by vladikoff** [comment]
Thanks Shane!

**Pull request merged**